### DISCUSSION TITLE
is there a way to config padding (top)?

### DISCUSSION THREAD

**Discussion by rahil627** [helix-editor/helix#11723] [Q&A]
in vs-code, the equivalent setting is `"editor.padding.top": 30`. it's no biggie, just a bit disturbing when things are on the edge.. particularly the first few lines of a file.

the only padding-related setting i've found is `scrolloff`

here's an example of wezterm on windows (with the title and tab bar off):

![image](https://github.com/user-attachments/assets/76b489a9-3c32-40eb-89c7-a17002717461)

hmm, i guess it's specifically the distance between the tab bar and main content, definitely helix territory, not the terminal

**Comment by jakobjpeters** [comment]
I also want this!
